clang/lib/CodeGen/CGExpr.cpp

```diff
@@ -1217,6 +1217,35 @@ void CodeGenFunction::EmitBoundsCheck(const Expr *E, const Expr *Base,
   EmitBoundsCheckImpl(E, Bound, Index, IndexType, IndexedType, Accessed);
 }
 
+llvm::DILocation *CodeGenFunction::SanitizerAnnotateDebugInfo(
```

Comment:
This good candidate for "NFC: SanitizerAnnotateDebugInfo" patch

Reply:
Spun off into https://github.com/llvm/llvm-project/pull/139965